src/postPage.js

```diff
@@ -0,0 +1,350 @@
+import "./styles/index.scss";
+
+function getLSNote(id) {
+  let data = localStorage.getItem('notes');
+  if(!data) {
+    data = [];
+    localStorage.setItem('notes', JSON.stringify(data));
```

Comment:
It's a safety measure in cases where the local storage value isn't initialized so I set it in just in case, although id doubt I would ever do that in modern react since everything has an initial implementation.
i guess i was trying to avoid undefined at all cost

Reply:
i also did think about decoupling the json logic from that function but tbh it was rushed and liked used only in one place so i was like nvm